docs/notes-KZG10.md

```diff
@@ -1,3 +1,53 @@
 In this module we show how and why the KZG10 
 polynomial commitment scheme has been modularised 
-for this PLONK implementation.
\ No newline at end of file
+for this PLONK implementation.
+
+KZG10 Commitments 
+==================
+
+PLONK can be constructed with different 
+commitment schemes and does not requre solely
+homomorphic commitments. However, this library
+has only homomorphic commitment schemes as they 
+are intelligble for users and have many useful
+properties.
+
+We use 'KZG10 commitments', often called 'Kate' 
+commitments refers to the commitments scheme 
+created by Kate, Zaverucha and Goldberg. 
```

Comment:
Add ref to the paper maybe?
https://www.iacr.org/archive/asiacrypt2010/6477178/6477178.pdf

Reply:
It is there but a bit further down. Although I do agree with you, it should be put in there.